frontendservice/src/views/CustomerHome.vue

```diff
@@ -7,48 +7,168 @@ CONDITIONS OF ANY KIND, either express or implied. See the License for the
+  <div v-if="!isLoggedIn">
+    <NavBarCustomer></NavBarCustomer>
+    <cookieinfo></cookieinfo>
+    <cookieackno></cookieackno>
+    <div id="searchBarWrap" style="margin: 20px 0">
+      <div class="field has-addons">
+        <div class="form-group has-feedback">
+          <i class="fa fa-search" style="font-size:32px"></i>
+          <input id="searchBar" class="input" type="text" placeholder="Find a shop" />
+        </div>
+      </div>
     </div>
-    <div v-if="!isLoggedIn">
-      <router-link to="/about">Back</router-link>
+    <div id="wrap" class="control">
+      <div class="select">
+        <select>
+          <option>Select option</option>
+          <option>Food</option>
+          <option>Furniture</option>
+          <option>Electronics</option>
+          <option>Stationary</option>
+          <option>Jewellery</option>
+          <option>Agriculture</option>
+        </select>
+      </div>
+    </div>
+    <div class="panel panel-default">
+      <div class="table-responsive">
+        <table class="table">
+          <thead>
+            <th>Shop Name</th>
+            <th>Shop Type</th>
+            <th>Address</th>
+            <th>Active Customers</th>
+          </thead>
+          <tbody>
+            <tr v-for="shop in shops" :key="shop.shopId">
+              <td>
+                <a>
+                  <router-link
+                    v-bind:to="{name: 'specificShop', params: {Id: shop.shop.shopId} }"
+                  >{{shop.shop.shopName}}</router-link>
+                </a>
+              </td>
+              <td>{{shop.shop.shopType}}</td>
+              <td>{{shop.shop.address}}</td>
+              <td>{{shop.numberOfActiveTokens}}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
     </div>
-    <br />
   </div>
 </template>
 
 <script>
 import firebase from "firebase";
-
+import NavBarCustomer from "@/components/NavBarCustomer";
+import cookieinfo from "@/components/cookieinfo";
+import cookieackno from "@/components/cookieackno";
+import axios from "axios";
+import Cookie from "js-cookie";
 export default {
   name: "customerHome",
-
+  components: {
+    NavBarCustomer,
+    cookieinfo,
+    cookieackno
+  },
   data() {
     return {
       isLoggedIn: false,
+      timer: "",
+      shops: []
     };
   },
-  created() {
-    var user = firebase.auth().currentUser;
-    if (user) {
-      this.isLoggedIn = true;
+
+  methods: {
+    apifunction() {
+      var user = firebase.auth().currentUser;
+      if (user) {
+        this.isLoggedIn = true;
+      }
+      const t = this;
+      axios
+        .get("http://penguin.termina.linux.test:8085/shops")
+        .then(function(res) {
+          t.shops = res.data.shops;
+        });
+    },
+
+    beforeDestroy() {
+      clearInterval(this.timer);
+    },
+
+    cancelAutoUpdate() {
+      clearInterval(this.timer);
     }
   },
+
+  created() {
+    this.apifunction();
+    this.timer = setInterval(this.apifunction, 1);
```

Comment:
Add comment after "1" describing the unit. e.g. 1 /* <unit_of_time_used> */

Reply:
yes added it.